commands/insult.js

```diff
@@ -0,0 +1,51 @@
+const { SlashCommandBuilder } = require('@discordjs/builders');
+const config = require('../config.json');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/EvilInsultGenerator/discord-bot/pullRequest?prid=8755514)
```suggestion
const config = require("../config.json");
```